src/components/Facility/PatientNotesList.tsx

```
@@ -1,101 +1,107 @@
+          {notes.map((note) => {
+            const noteCard = (
+              <PatientNoteCard
+                note={note}
+                refetch={refetch}
+                disableEdit={disableEdit}
+                setReplyTo={setReplyTo}
+                mode={mode}
+                allowThreadView={mode === "thread-view"}
+                allowReply={mode !== "thread-view"}
+                setThreadViewNote={setThreadViewNote}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)